doc/dsl/interpretation_laufzeit.md

```diff
@@ -1,13 +1,323 @@
 ---
-title: "Interpreter Laufzeit"
+title: "Interpretation & Laufzeit"
 ---
 
 ## Überblick: Wie funktioniert die Interpretation allgemein
 
-## `MemorySpace`s und `Value`s
+Der `DSLInterpreter` muss zur Interpretation eines DSL Programms Ausdrücke evaluieren und die evaluierten Werte
+speichern. Dies gilt bspw. auch für die Member einer `quest_config`-Definition.
+
+Die Schritte, welche anschließend während der Interpretation vom `DSLInterpreter` ausgeführt
+werden, sind im folgenden Diagram dargestellt:
+
+![UML: Interpretatiospipeline](img/interpretation_pipeline.png){width="50%"}
+
+Die Interpretations-Pipeline ist in zwei Phasen aufgeteilt, die Laufzeitinitialisierung und die Interpretation.
+
+### Laufzeitinitialisierung
+
+Zuerst lädt der `DSLInterpreter` die Symbol- und Typinformationen aus der übergebenen `IEnvironment`-Instanz
+in ein `RuntimeEnvironment`.
```

Comment:
Hier fehlt noch was zum Verständnis. Wo kommt die übergebene `IEnvironment`-Instanz her? Warum wird sie in eine `RuntimeEnvironment` geladen (von den Namen her sind beides Environments)? Was machen diese Strukturen (grob im Überblick)?

Reply:
Habe den Text dahingehend angepasst